### PR TITLE
Fix sshd.pid access denied

### PR DESCRIPTION
**Pull request by ghost**
Create pid folder with proper ACL
Create sshd.pid with proper ACL

### CODE REVIEW

**Review by manojampalam — CHANGES_REQUESTED** [review]
Thanks for your contribution. sshd pid logic for Windows is currently flawed. the child worker typically overwrites the pid entry making this file useless. Please disable the code around writing pid for Windows for now ($ifdef out).

**Comment by ghost** [comment]
I will revise this to disable the pid creation in code instead.

File: contrib/win32/openssh/install-sshd.ps1 [hunk 1]
```diff
@@ -299,4 +307,11 @@ $accessRule = New-Object System.Security.AccessControl.FileSystemAccessRule($ssh
 $acl.SetAccessRule($accessRule)
 Set-Acl -Path $sshdlog -AclObject $acl
 
+$sshdpid = Join-Path $piddir "sshd.pid"
+if(-not (test-path $sshdpid)){ $null | Set-Content $sshdpid }
+$rights = [System.Security.AccessControl.FileSystemRights]"Read, Write"
+$accessRule = New-Object System.Security.AccessControl.FileSystemAccessRule($sshdAccount, $rights, "None", "None", "Allow")
+$acl.SetAccessRule($accessRule)
+Set-Acl -Path $sshdpid -AclObject $acl
```
Comment (bagajjal) [inline contrib/win32/openssh/install-sshd.ps1:315]: I don't think there is a need for setting ACL on the sshd.pid file..

fyi, we set the ACL on log files (sshd.log, ssh-agent.log) as they contain sensitive information.

**Comment by manojampalam** [comment]
This issue is now fixed with recent changes to overall architecture. SSHD now runs as system and the listener logs its PID at %programfiles%\ssh

**Pull request closed without merging**